launch/bringup.launch

```diff
@@ -0,0 +1,64 @@
+<!--
+Copyright (c) 2020-2021 Pilz GmbH & Co. KG
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-->
+
+<!-- Use this launch-file for starting the scanner driver exclusively. -->
+<launch>
+  <!-- IP-Address of Safety laser scanner -->
+  <arg name="sensor_ip" default="192.168.0.10" />
+
+  <!-- Prefix i. e. name of the scanner (required to run multiple scanners) -->
+  <arg name="prefix" default="laser_1" />
+
+  <!-- Start angle of measurement in radian -->
+  <arg name="angle_start" default="$(eval radians(-137.4))" />
+
+  <!-- End angle of measurement in radian -->
+
+  <arg name="angle_end" default="$(eval radians(137.4))" />
+
+  <!-- Publishing of intensities -->
+  <arg name="intensities" default="false" />
+
+  <!-- Scan resolution in radian -->
+  <arg name="resolution" default="$(eval radians(0.1))" />
+
+  <!-- IP-Address of host machine -->
+  <arg name="host_ip" default="auto" />
+
+  <!-- UDP Port on which monitoring frames (scans) should be received -->
+  <arg name="host_udp_port_data" default="55115" />
+
+  <!-- UDP Port used to send commands (start/stop) and receive the corresponding replies -->
+  <arg name="host_udp_port_control" default="55116" />
+
+  <!-- Set the following to true in order to publish scan data as soon as a UDP packet is ready, instead of waiting for a full scan -->
+  <arg name="fragmented_scans" default="false" />
+
+  <node name="$(arg prefix)_scanner" type="psen_scan_v2_node" pkg="psen_scan_v2" output="screen" required="true">
```

Comment:
Suggestion: Could the suffix be dropped?
```suggestion
  <node name="$(arg prefix)" type="psen_scan_v2_node" pkg="psen_scan_v2" output="screen" required="true">
```

Reply:
I think having an arg `prefix` which actually ends up beeing the `node_name` is strange. The word is **pre**fix.

Reply:
Ok, but `laser_1_scanner` is also strange. How about using the prefix as namespace, like for the prbt? Then we could name it
`laser_1/scanner` or
`laser_1/driver` or
`scanner_1/driver`
What do you think?

Reply:
Well you could choose another prefix if 'laser_1_scanner' sounds strange. Maybe 'first_laser_scanner'? 
Imho using a namespace is ok as well. Would still prefer the word 'scanner' then.

Reply:
how about the suffix being `_node`.
then it would be `laser_1_node`

Reply:
and for the tf frames it is still a prefix. @agutenkunst

Reply:
Using Christians proposal now. `<prefix>_node` = `laser_1_node`

Reply:
Like this you still have different names for tf an node namespace?! I think, the idea was to have unique names??

Reply:
@martiniil maybe you could state in the description how setting the prefix influences:

- topicname
- tf name
- node name

Its hard to keep track of all the implications...

Reply:
see description ...

Reply:
This sounds good to me! Would merge